src/main/java/reactor/netty/Metrics.java

```diff
@@ -20,6 +20,18 @@
 import java.net.SocketAddress;
 
 public class Metrics {
+	public static final String HTTP_SERVER_NAME_PREFIX = "reactor.netty.http.server";
+
+	public static final String HTTP_CLIENT_NAME_PREFIX = "reactor.netty.http.client";
+
+	public static final String TCP_SERVER_NAME_PREFIX = "reactor.netty.tcp.server";
+
+	public static final String TCP_CLIENT_NAME_PREFIX = "reactor.netty.tcp.client";
+
+	public static final String UDP_SERVER_NAME_PREFIX = "reactor.netty.udp.server";
+
+	public static final String UDP_CLIENT_NAME_PREFIX = "reactor.netty.udp.client";
+
```

Comment:
These look like could be shortened a bit, minus the `_NAME_` part. `Metrics.HTTP_SERVER_PREFIX` still reads pretty well to me with "prefix" and the type `String` + some Javadoc perhaps could be a good balance between readability and brevity.

Reply:
done 69ac2d5607282f3e95320f40ce6f12362790792a